cubic-server/protocol/serialization/popPrimaryType.hpp

```diff
@@ -215,25 +220,44 @@ constexpr std::string popChat(uint8_t *&at, uint8_t *eof) { return popString(at,
 // {
 // }
 
+static inline size_t _readMem(void *pck, uint8_t *data, size_t size)
```

Comment:
A static function in a `.hpp` serves no purpose, as the function can always be accessed no matter what from any file that includes `popPrimaryType.hpp`